### PR TITLE
[CARBONDATA-1812] Provide API to get table dynamic information(table size and last modified time)

### PR DESCRIPTION
**Pull request by QiangCai**
provide API to get table dynamic information(table size and last modified time)	
 - [x] Any interfaces changed?
 new interface,  don't change old interface
 - [x] Any backward compatibility impacted?
 no
 - [x] Document update required?
 no
 - [x] Testing done
        Please provide details on 
        - Whether new unit test cases have been added or why no new tests are required?
           added
        - How it is tested? Please attach test report.
        - Is it a performance related change? Please attach the performance test report.
        - Any additional information to help reviewers in testing this change.
       
 - [x] For large changes, please consider breaking it into sub-tasks under an umbrella JIRA. 
 small change

### CODE REVIEW

File: integration/spark2/src/main/scala/org/apache/spark/sql/execution/command/schema/GetTableDetailCommand.scala [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+import org.apache.carbondata.core.constants.CarbonCommonConstants
+import org.apache.carbondata.core.metadata.schema.table.CarbonTable
+import org.apache.carbondata.core.metadata.AbsoluteTableIdentifier
+import org.apache.carbondata.core.statusmanager.SegmentStatusManager
+import org.apache.carbondata.core.util.{CarbonProperties, CarbonUtil}
+
+/**
+ * collect dynamic detail information of the table, including table size, last modified time, etc.
+ */
+case class GetTableDetailCommand(
```
Comment (jackylk) [inline integration/spark2/src/main/scala/org/apache/spark/sql/execution/command/schema/GetTableDetailCommand.scala]: Add Carbon prefix

File: integration/spark2/src/main/scala/org/apache/spark/sql/execution/command/schema/GetTableDetailCommand.scala [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+        getLastModifiedTime(absoluteTableIdentifier)
+      )
+    }
+  }
+
+  private def getLastModifiedTime(absoluteTableIdentifier: AbsoluteTableIdentifier): Long = {
+    SegmentStatusManager.getTableStatusLastModifiedTime(absoluteTableIdentifier)
+  }
+
+  private def getSizeInBytes(
```
Comment (jackylk) [inline integration/spark2/src/main/scala/org/apache/spark/sql/execution/command/schema/GetTableDetailCommand.scala]: Add this in CarbonTable class

**Comment by ravipesala** [comment]
SDV Build Success , Please check CI http://144.76.159.231:8080/job/ApacheSDVTests/1901/

**Comment by CarbonDataQA** [comment]
Build Success with Spark 2.1.0, Please check CI http://136.243.101.176:8080/job/ApacheCarbonPRBuilder1/1484/

**Comment by CarbonDataQA** [comment]
Build Failed  with Spark 2.1.0, Please check CI http://136.243.101.176:8080/job/ApacheCarbonPRBuilder1/1487/

**Comment by CarbonDataQA** [comment]
Build Success with Spark 2.1.0, Please check CI http://136.243.101.176:8080/job/ApacheCarbonPRBuilder1/1491/

**Comment by ravipesala** [comment]
SDV Build Success , Please check CI http://144.76.159.231:8080/job/ApacheSDVTests/1905/

**Comment by ravipesala** [comment]
SDV Build Fail , Please check CI http://144.76.159.231:8080/job/ApacheSDVTests/1909/

**Pull request closed without merging**